shell/package.json

```diff
@@ -204,8 +205,7 @@
     "webpack-merge": "^5.7.3"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx,json,scss,css,md}": "prettier --write",
-    "app/common/**/*.{js,jsx,ts,tsx}": "npm run check-unit-test"
+    "*.{js,jsx,ts,tsx,json,scss,css,md}": "prettier --write"
```

Comment:
Removed check ?

Reply:
remove temporary, check logic needs to be adjusted